lib.sh

```diff
@@ -370,7 +370,8 @@ get_newest_dat_files() {
             print_text_in_color "$ICyan" "Downloading new IPv4 dat file..."
             sleep 1
             check_command curl_to_dir "$GEOBLOCKDAT" "$IPV4_NAME" "$SCRIPTS"
-            check_command rm /usr/share/GeoIP/GeoIP.dat
+            mkdir -p /usr/share/GeoIP
+            rm /usr/share/GeoIP/GeoIP.dat
```

Comment:
Please don't assume that the file is there, it will produce an error if not `-f` is used.